### PR TITLE
Linux

### PR DESCRIPTION
**Pull request by konserw**
Hello,
These are changes I've made to use this on linux (thank you for vary much for that btw. :) ).
Main change is case sensitive spelling of header to be included in EEKeeperWindow.ui + when editing it QtCreator asked about correct path to the qrc file which I've provided. 
Apart from that I've removed your qtcreator settings file and added gitignore to prevent any generated files to be uploaded.
Best regards, konserw

### CODE REVIEW

**Comment by devurandom** [comment]
1. The code comes from "Troodon80" - I just imported it to GitHub.
    - In case you want to try to find him: He probably also goes by the name "J. Beadle", judging from the name of his home directory, as mentioned in the EEKeeper.pro.user file.
2. I don't see a change to headers included in EEKeeperWindow.ui - could you please point me to that?
3. In EEKeeperWindow.ui you also changed the height of a widget. Was that intentional?
4. I see a lot of changes to the include path of qrc files, which appears to be correct.
5. Removal of the EEKeeper.pro.user file and addition of .gitignore seems reasonable.

If you address 2 and 3 and provide a more complete commit message, I'll merge your changes.

File: EEKeeper/ui/linux/EEKeeperWindow.ui [hunk 1]
```diff
@@ -418,9 +418,14 @@
    <header>SpellBrowserWidget.h</header>
    <container>1</container>
   </customwidget>
+  <customwidget>
+   <class>ItemBrowserWidget</class>
+   <extends>QWidget</extends>
+   <header>ItemBrowserWidget.h</header>
```
Comment (konserw) [inline EEKeeper/ui/linux/EEKeeperWindow.ui]: This is the header change

File: EEKeeper/ui/linux/EEKeeperWindow.ui [hunk 2]
```diff
@@ -75,7 +75,7 @@
      <x>0</x>
      <y>0</y>
      <width>798</width>
-     <height>22</height>
+     <height>25</height>
```
Comment (konserw) [inline EEKeeper/ui/linux/EEKeeperWindow.ui]: I believe it was done by qtcreator itself

**Comment by konserw** [comment]
Is explanation for 2. and 3. sufficient? What else would you like to see in commit message?
Are you planing to maintain this project? Ifso I could send some other PRs.

**Comment by devurandom** [comment]
A commit message should sufficiently summarise the changes made, such that I can scroll through the commit log and know what to expect from each commit without looking at the patch - e.g. when I am searching for a certain problem or a certain type of change. The general format I use is:
```
One line summary of the intention

More detailed explanations of different changes and the reasons I chose this way over other possibilities.
```

I am not planning to develop EEKeeper myself. When I created this repository, I was searching for savegame editors, found very few to be available with source code, so I tried to preserve those I found. If you want to take over maintainership, I'd be glad to point people to your fork.

**Comment by konserw** [comment]
I've reworded commit messages. I hope now they are informative enough.
As for the future of this project - I think the main question is what does uthor think of it. And if anyone else beside two of us knows about it :) 
To be honest I've only found this because it was more fun for me to compile eekeeper-qt on linux then to power on windows VM for the official eekeeper build for windows.

**Comment by Troodon80** [comment]
Sorry for the late reply here. I actually replied to the e-mail and forgot about this discussion.

I don't have much to add. Either I can upload what I have as my own project (last thing I did was get the spell and item browsers working), or take ownership of the project you currently have here and commit, or konserw can take ownership.

At this point, I have been too busy with work to focus primarily on the Qt version and I have been more focused on simply maintaining the version which is fully functional.

A few months ago I overhauled the MFC version from the ground up for better file handling and memory management (the initial loading of resources is now almost instant, so much so that someone on the Beamdog forum thought it was a bug) to the point where there is no comparison between it and the Qt version.

If people are willing to help develop the Qt version of Keeper, I will give what time I can (it's not much) to its completion. :-)

**Comment by konserw** [comment]
I missed email from you somehow, so good that you replied here ;)
I would create eekeeper github "organization" account with both mfc and qt version repositories (see here: https://help.github.com/articles/creating-a-new-organization-from-scratch/ to create something like this: https://github.com/cucumber ). Probably @devurandom could then transfer this repository there, you could add mfc repository and ad all 3 of us as admins for that organization? 
Thank you for trust, but I don't think I could be owner/maintainer of this project alone. I know some Qt/C++ but doesn't know BG inner workings at all...

**Comment by Troodon80** [comment]
Well, unfortunately, like I told @devurandom over on [SourceForge](https://sourceforge.net/p/eekeeper/tickets/43/), the MFC version has code in it that I can't release to the public for various reasons (not related to copyright or licence). So I won't be adding that (at least not yet). I can, however, create a new repository (or organisation) for just the Qt version.

**Comment by konserw** [comment]
It's a pity since what you said Qt version could benefit from changes you made in mfc. But if it's not related to copyright/license maybe you could opensource it in the future under that account. 
All in all you are author so probably you should decide

**Comment by devurandom** [comment]
Hello @Troodon80, great to see you here! I just initiated a repository transfer to you. If you would accept that, you could merge/rebase your changes into it. I think that's the best solution, since you're the original author and I was just this code's keeper. If @konserw has more additions, I suggest he submits pull requests against your repository. Or, if he one day feels confident enough, takes over maintainership entirely.

**Comment by konserw** [comment]
It's also a good solution 👍

**Pull request merged**